wordle-web/app.vue

```diff
@@ -1,46 +1,79 @@
 <template>
-  <NuxtLayout>
-    <v-app>
-      <v-app-bar color="primary" :elevation="2">
-        <template v-slot:prepend>
-          <v-icon color="secondary"> mdi-book </v-icon>
-        </template>
-
-        <v-app-bar-title>WordleApp</v-app-bar-title>
-
-        <v-btn @click="router.push('/')">Home</v-btn>
-        <v-btn @click="router.push('/test')">Test</v-btn>
-        <v-btn icon="mdi-theme-light-dark" @click="toggleTheme" />
-        <v-btn icon="mdi-help-circle" @click="showHelpDialog = true" />
-        <HelpDialog v-model="showHelpDialog" />
-      </v-app-bar>
-      <v-main>
-        <NuxtPage />
-      </v-main>
-    </v-app>
-  </NuxtLayout>
+    <NuxtLayout>
+        <v-app>
+            <v-app-bar color="primary" :elevation="2">
+                <template v-slot:prepend>
+                    <v-btn icon @click="router.push('/')">
+                        <v-icon color="deep-purple">mdi-wizard-hat</v-icon> <!-- Wizard hat icon -->
+                    </v-btn>
+                    <v-btn @click="router.push('/')">
+                        Welcome to LexiQuest
+                    </v-btn>
+                </template>
+
+                <v-spacer></v-spacer> <!-- This pushes the rest of the items to the right -->
+
+                <v-btn icon="mdi-cog" @click="toggleSettingsDrawer" />
+                <v-btn icon="mdi-help-circle" @click="showHelpDialog = true" />
+                <v-btn icon="mdi-menu" @click="toggleDrawer" />
+
```

Comment:
We created a dialogue for the player name and added that to our components folder. This resource helped me understand the "display the name as the user types" requirement! https://vuejs.org/guide/essentials/forms